test/cli/test-mutating-admission-policy/match-condition/kyverno-test.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kyverno.io/v1
+kind: Test
+metadata:
+  name: test-check-match-condition
+policies:
+- policy.yaml
+resources:
+- resource1.yaml
+- resource2.yaml
+results:
+- policy: set-replicas-color-red
+  resource: nginx-red
+  kind: Deployment
+  result: pass
+- policy: set-replicas-color-red
+  resource: nginx-no-color
+  kind: Deployment
+  result: skip
```

Comment:
The following fields are missing in each test result:
1. `isMutatingAdmissionPolicy` should be set to `true`.
2. `patchedResources` should be set to the path of the mutated resource. If there is no mutated resource, we shouldn't add it.